apps/datahub/src/app/record/navigation-bar/navigation-bar.component.html

```diff
@@ -23,26 +42,42 @@
         <button
           (click)="setActiveLabel($event.target)"
           gnUiAnchorLink="{{ l.anchor }}"
-          gnUiAnchorLinkDisabledClass="cursor-default opacity-60"
-          class="block px-3 py-3 rounded-md font-medium uppercase tracking-wider w-full text-left truncate"
+          gnUiAnchorLinkDisabledClass="hidden"
+          class="h-[60px] block px-3 py-3 hover:bg-primary hover:bg-opacity-10 font-medium w-full text-left text-wrap"
           translate
         >
           {{ l.label }}
         </button>
       </div>
     </div>
     <button
-      class="block px-3 py-3 rounded-md font-medium uppercase tracking-wider shrink truncate sm:hidden"
+      class="h-[60px] block px-3 py-3 rounded-md font-medium shrink text-wrap sm:hidden"
       [class]="displayMobileMenu ? 'hidden' : 'block'"
       translate
     >
       {{ activeLabel }}
     </button>
-    <button
-      type="button"
-      class="h-10 inline-flex items-center justify-center p-2 rounded-md shrink-0 sm:hidden"
-    >
-      <ng-icon class="align-middle" name="matExpandMoreOutline"></ng-icon>
-    </button>
+  </div>
+  <div
+    class="h-[60px] flex flex-row gap-3 min-w-40 justify-center items-center lg:min-w-60 lg:pl-4 lg:justify-start"
+    [class]="displayMobileMenu ? 'border-l-2 border-gray-300' : ''"
```

Comment:
Does it work with `[class]`? I thought it needed ngClass:

```suggestion
    [ngClass]="displayMobileMenu ? 'border-l-2 border-gray-300' : ''"
```

Reply:
Yep, double-checked and it works:-)